Branch-SDK/src/main/java/io/branch/referral/ServerRequestCreateUrl.java

```diff
@@ -22,7 +22,6 @@
     private Branch.BranchLinkCreateListener callback_;
     /* Default long link base url*/
     private static final String DEF_BASE_URL = "https://bnc.lt/a/";
-    private boolean isReqStartedFromBranchShareSheet_;
```

Comment:
As mentioned in the previous comment, this is not used anywhere, plus we have a few overloaded methods that `set` this value but, again, we never `get` it, so don't need it and we don't need the overloaded methods.